src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -8,7 +8,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "iManager has been cleared!";
+    public static final String MESSAGE_SUCCESS = "Tasklist has been cleared!";
```

Comment:
lol actually this is English, so should be Task list? Though I suppose it's reasonable to put it like this too?